domains/client/domain-operator/src/tests.rs

```diff
@@ -808,6 +812,104 @@ async fn test_finalize_block_proof_creation_and_verification_should_work() {
     test_invalid_state_transition_proof_creation_and_verification(3).await
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn test_false_invalid_state_transition_proof_is_rejected() {
```

Comment:
Perhaps `test_bad_invalid_state_transition_proof_is_rejected` to not mixing up with the invalid bundle fraud proof